File: Java/Lesson_Three/Nested.java

/**
 * Write a description of class Nested here.
 * 
 * @author (your name) 
 * @version (a version number or a date)
 */
public class Nested
{
    public static void main(String [] args){
        //nesting if-else statements
        
        boolean boolt = true;
        boolean boolf = false;
        
        //if if
        if(boolt){
            if(boolt){
                System.out.println("This is inside two if's");
            }
            else{
                
            }
        }
        else{
            
        }
        
        //else if
        if(boolf){
            
        }
        else{
            if(boolt){
                System.out.println("This is inside and else, then an if");
            }
            else{
                
            }
        }
        
        //if else
        if(boolt){
            if(boolf){
                
            }
            else{
                System.out.println("This is inside an if then an else");
            }
        }
        else{
            
        }
        
        //else else
        if(boolf){
            
        }
        else{
            if(boolf){
                
            }
            else{
                System.out.println("This is inside of two else's");
            }
        }
    }
}

File: Python/Lesson-Three/Nested.py
boolt = True
boolf = False

if boolt:
	if boolt:
		print "This is inside two ifs"
	else:
		print "This won't print"
else:
	print "This also won't print"



if boolf:
	print "This won't print"
else:
	if boolt:
		print "This will print inside an else then and if"
	else:
		print "This will not print"


if boolt:
	if boolf:
		print "This won't print"
	else:
		print "This will print inside an if then and else"
else:
	print "This won't print"


if boolf:
	print "This won't print"
else:
	if boolf:
		print "This won't print"
	else:
		print "This will print inside an else then an else"
